CONTRIBUTING.md

```diff
@@ -27,11 +36,11 @@ Frontend dle [Angular Style Guide](https://angular.io/guide/styleguide).
 
 ## Pravidla přispívání
 - kód **v angličtině**,
-- commity **v angličtině**,
+- commity **v angličtině** (+ commit messages ideálně podle [Conventional Commits styleguide](https://www.conventionalcommits.org/en/v1.0.0/)),
```

Comment:
Tohle je otevrena otazka v tech-leads channelu, precetl jsem si vyhody a jedina, ktera je pro nas relevantni je "Novi prispevatele se snadno zorientuji v historii" (1 z 5). Osobne nejsem pro zavadeni procesu bez silne potreby a zatim me o tehle Styleguide nic nepresvedcilo.

Nebranim se tomu a rad si poslechnu argumentaci pro ve Slacku.

Reply:
napsal jsem to takhle, aby tam byla nějaká guidance, ale ne nutně proces jako takový

nemám vůbec problém buď to vyhodit nebo se tam dostat postupně

Reply:
pro teď jsem to vyhodil - dokud se na tom neshodneme